distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -442,6 +443,64 @@ type Number
      floating point numbers.
 @Builtin_Type
 type Float
+    ## GROUP Conversions
+       ICON convert
+       Generic conversion of an arbitrary Enso value to requested type.
```

Comment:
_arbitrary Enso value_!? I guess the whole documentation has just been copied. The documentation doesn't make sense here.

Reply:
This is removed.